modules/twinklefluff.js

```diff
@@ -586,8 +582,14 @@ Twinkle.fluff.init = function twinklefluffinit() {
 
 		if ( Morebits.queryString.exists( 'twinklerevert' ) ) {
 			Twinkle.fluff.auto();
-		} else {
-			Twinkle.fluff.normal();
+		} else if( mw.config.get('wgNamespaceNumber') === -1 && mw.config.get('wgCanonicalSpecialPageName') === "Contributions" ) {
+			Twinkle.fluff.contributions();
+		} else if( mw.config.get('wgDiffOldId') || mw.config.get('wgDiffNewId') ) { // wgDiffOldId included for clarity in if else loop [[phab:T214985]]
```

Comment:
That is, `wgDiffNewId` alone is sufficient to determine if we're on a diff page, but I've included `wgDiffOldId` here to the following `else if` a little simpler.  Plus, this should be a little clearer to maintain, especially if that ever changes.